radix-engine/src/blueprints/package/package.rs

```diff
@@ -209,7 +215,12 @@ impl PackageNativePackage {
     pub fn schema() -> PackageSchema {
         let mut aggregator = TypeAggregator::<ScryptoCustomTypeKind>::new();
 
-        let substates = Vec::new();
+        let mut substates = Vec::new();
+        substates.push(aggregator.add_child_type_and_descendents::<PackageInfoSubstate>());
```

Comment:
These were somehow missed off the schema... I guess we don't validate these yet?

Reply:
This is probabaly caused by `PackagePackage` creating packages via `KernelApi` rather than `SystemApi` (which enforces payload-schema check).

Reply:
Very possibly! It also turns out we didn't validate that someone can't write substates at offsets above their schema. Which this PR fixes :).

In a follow-up PR I'm going to add a test that all genesis substates match their respective schemas